docs/config/ping-federate-example.md

```diff
@@ -35,121 +41,148 @@ For example:
+1. Sign in to your PingFederate instance as a user with administrative privileges.
+2. Navigate to **SP Connections** and select **Create New**.
+3. Ensure the **Browser SSO Profiles** connection template with **Protocol SAML 2.0** is selected and then click **Next**.
+4. On the **Connection Options** tab ensure only **Browser SSO** is selected.
 
-7. After clicking **Save**, navigate to the **Mappers** tab and select **Create**.
-8. Make the following configurations:
+5. On the **General Info** tab 
+  1. In **Partner’s Entity Id** add the value given for **entityID** in the **certificate descriptor API**.
+  2. Add a connection name.
+  3. In Base URL, add $ELBADDRESS.
+6. On the **Browser SSO** tab click **Configure Browser SSO** which will launch a new set of tabs for configuring the browser SSO.
 
-   * Enter *email* into **Name**.
-   * Choose *Attribute Importer* from the **Mapper Type** dropdown.
-   * Enter *EMAIL* into **Attribute Name**.
-   * Enter *email* into **User Attribute Name**.
+   #### Configuring browser SSO
+1. On the **SAML Profiles** tab tick all four checkboxes.
+2. On the **Assertion Creation** tab click **Configure Assertion Creation** which will launch a new set of tabs for configuring the assertion creation.
+
+  #### Configuring assertion creation
+  1. On the **Identity Mapping** tab ensure the **Standard** checkbox is ticked.
+  2. Under the heading **Attribute Contract**:
+     1. Enter *Email* under **Extend the contract**.
+     2. Choose *urn:oasis:names:tc:SAML:2.0:attrname-format:basic* from the **Attribute name format** dropdown and click **Add**.
+     3. Enter *FirstName* under **Extend the contract**.
+     4. Choose *urn:oasis:names:tc:SAML:2.0:attrname-format:basic* from the **Attribute name format** dropdown and click **Add**.
+     5. Enter *LastName* under **Extend the contract**.
+     6. Choose *urn:oasis:names:tc:SAML:2.0:attrname-format:basic* from the **Attribute name format** dropdown and click **Add**.
+  3. On the **Authentication Source Mapping** tab click **Map New Adapter Instance...** which will launch a new set of tabs for mapping a new adapter instance.
+
+  #### Mapping a new adapter instance
+ 1. On the **Adapter Instance** tab select *IdP Adapter* from the dropdown menu.
+ 2. On the **Attribute Contract Fulfilment** tab:
+     1. In the **Email** row select *Adapter* from the **source** dropdown.
+     2. In the **Email** row select *email* from the **value** dropdown.
+     3. In the **FirstName** row select *Adapter* from the **source** dropdown.
+     4. In the **FirstName** row select *fname* from the **value** dropdown.
+     5. In the **LastName** row select *Adapter* from the **source** dropdown.
+     6. In the **LastName** row select *lname* from the **value** dropdown.
+     7. In the **SAML_SUBJECT** row select *Adapter* from the **source** dropdown.
+     8. In the **SAML_SUBJECT** row select *subject* from the **value** dropdown.        
```

Comment:
This numeration is rather confusing. Suppose a user reports a problem with "step 2". What would it refer to?

For sub-steps of step 2, don't number them at all, use letters or use 2.1, 2.2, etc.